resources/param.rb

```diff
@@ -27,8 +27,6 @@
     '/etc/sysctl.conf.local'
   when 'suse'
     '/etc/sysctl.conf' if node['platform_version'].to_f < 12.0
-  else
-    nil
```

Comment:
why the nil removal? 

I guess it still works 👍

Reply:
Case statements default to nil if there's no match:
irb#1(main):001:0> platform_family = 'bob'
=> "bob"
irb#1(main):006:0> case platform_family
irb#1(main):007:1>   when 'freebsd'
irb#1(main):008:1>     '/etc/sysctl.conf.local'
irb#1(main):009:1>   when 'suse'
irb#1(main):010:1>     '/etc/sysctl.conf' if node['platform_version'].to_f < 12.0
irb#1(main):011:1> end
=> nil